.circleci/config.yml

```diff
@@ -50,14 +40,17 @@ jobs:
       - image: circleci/python:3.9
         <<: *with-requests
     <<: *job-defaults
-
+  python-3_10:
+    docker:
+      - image: circleci/python:3.10
+        <<: *with-requests
+    <<: *job-defaults
 workflows:
   main:
     jobs:
       - lint-sdk
-      - python-3_4
-      - python-3_5
```

Comment:
I think we agreed @bartes that we need to support at least python 3.5 as it's default in ubuntu bionic which is still supported.

Reply:
👍